theme/ItaliaTheme/Blocks/_argumentsInEvidence.scss

```diff
@@ -12,6 +12,14 @@
     background-color: $primary;
   }
 
+  .card {
```

Comment:
non mettiamola qui la regola. 
Mettiamola in nel file che sovrascrive le regole di boostrap italia per le card (bootstrap-override/bootstrap-italia/card)
e sovrascriviamo (con la stessa alberatura di classi di https://github.com/italia/bootstrap-italia/blob/1640f1b389ff39b1481434c3ec7a4b998927a5ec/src/scss/custom/_card.scss) a.read-more.
Se sono stata troppo contorta con la spiegazione sentiamoci :-D